src/components/UI/app_details/body.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+const DescriptionList = styled.dl({
+  margin: 0,
+
+  dd: {
+    margin: 0,
+    ul: {
+      marginBottom: 15,
+      padding: 0,
+      listStyleType: 'none',
```

Comment:
According to what we discussed on Slack, this would have to be `marginBottom: '15px'` etc., no?

Reply:
Didn't know we agreed on that. Will change it